9.mAP/notebooks/4.generate_map_scores_morphology_treatment.ipynb

```diff
@@ -1,1193 +1,5039 @@
 {
```

Comment:
<p>Forgot mention this in earlier notebooks. Did these function perform what it was supposed to do? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Yes!</p>